src/main/java/org/folio/innreach/repository/MARCTransformationOptionsSettingsRepository.java

```diff
@@ -2,10 +2,17 @@
 
 import org.folio.innreach.domain.entity.MARCTransformationOptionsSettings;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.QueryHints;
 import org.springframework.stereotype.Repository;
 
+import javax.persistence.QueryHint;
+import java.util.Optional;
 import java.util.UUID;
 
 @Repository
 public interface MARCTransformationOptionsSettingsRepository extends JpaRepository<MARCTransformationOptionsSettings, UUID> {
+  @QueryHints({ @QueryHint(name = org.hibernate.annotations.QueryHints.PASS_DISTINCT_THROUGH, value = "false") })
+  @Query(value = "SELECT DISTINCT m FROM MARCTransformationOptionsSettings AS m where m.centralServer.id = :id")
```

Comment:
As I remember we place query definition into entity and make a reference to it from repository.